packages/perseus/src/widgets/interactive-graphs/graphs/components/svg-line.tsx

```diff
@@ -20,7 +20,7 @@ export function SVGLine(props: Props) {
             x2={end[0]}
             y2={end[1]}
             style={style}
-            className={`${className}`}
```

Comment:
This was making svg `<line>` have `class="undefined"`. Now it should have no class name if one isn't passed in.